setup.cfg

```diff
@@ -1,16 +1,15 @@
 [metadata]
-description-file = README.rst
-# name = slsim
-# author = DESC/SLSC
-# author_email =
-# description = A package for simulating strong lenses for LSST.
-# long_description = file: README.rst
-# long_description_content_type = text/x-rst
-# license = MIT
-# license_file = LICENSE.rst
-# url = https://github.com/LSST-strong-lensing/slsim
-# edit_on_github = True
-# github_project = LSST-strong-lensing/slsim
+name = slsim
```

Comment:
you seem to have reverted the changes I made to make it compatible with the pypi displaying

Reply:
I have pulled your changes now.